### ISSUE TITLE
Swagger Facet Ignored in enunciate.xml

### ISSUE THREAD

**Issue by tbartolo** [stoicflame/enunciate#964]
I am running enunciate 2.8 via Gradle. I am trying to force a particular API class to not be included when generating Swagger documentation. I am trying to mimic the example code given here: [https://github.com/stoicflame/enunciate/wiki/Facets](url). 

For some reason, the facet tag within the Swagger tag is being completely ignored and Swagger documentation is still being generated for endpoints which should not be. Please view my enunciate.xml file, along with the source code.

 **enunciate.xml**

```
<enunciate xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
           xsi:noNamespaceSchemaLocation="http://enunciate.webcohesion.com/schemas/enunciate-2.0.0.xsd">

    <title>MyService</title>
    <description>MyService description</description>
    <copyright>2019</copyright>
    <api-classes>
    	<include pattern="com.company.search.adminservice.api.**" />
    </api-classes>
    <modules>
        <docs disabled="false"> </docs>
		<jaxrs groupBy="path">
            <application path="/api"/>
        </jaxrs>
        <c-xml-client disabled="true"></c-xml-client>
        <gwt-json-overlay disabled="true"></gwt-json-overlay>
        <php-xml-client disabled="true"></php-xml-client>
		<ruby-json-client disabled="true"></ruby-json-client>
		<php-json-client disabled="true"></php-json-client>
		<obj-c-xml-client disabled="true"></obj-c-xml-client>
		<csharp-xml-client disabled="true"></csharp-xml-client>
        <jackson1 disabled="true"/>
        <swagger  disabled="false" basePath="/api">
        	<facets>
      	    	<exclude name="company.com/facets#disableswagger"/>	
        	</facets>
        </swagger>
    </modules>
</enunciate>
```

**custom annotation**
```
package com.company.search.commons.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;
import com.webcohesion.enunciate.metadata.Facet;

/**
 * Annotation to disable Swagger UI generation
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.METHOD)
@Facet("company.com/facets#disableswagger")
public @interface DisableSwagger {

}
```

**endpoint which should be ignored but is still generating Swagger**
```
@DisableSwagger
@PreAuthorize(Permission.ReadWrite)
@RequestMapping(value = "clearcache", method = RequestMethod.GET)
public Object moveClient(
         @RequestParam(name = "client", required = true) String clientName,
	 @RequestParam(name = "srcServiceName", required = true) String srcServiceName,
	 @RequestParam(name = "destServiceName", required = true) String destServiceName) {

		return clientConfigService.moveClientServiceConfiguration(clientName, ClusterConfigKeys.ServiceCode,
				srcServiceName, destServiceName);
	}
```

**Comment by stoicflame** [comment]
Bug. Fixed at 72dd0f5c5.

**Comment by stoicflame** [comment]
[Enunciate 2.12](https://github.com/stoicflame/enunciate/releases/tag/v2.12.0) has been released.
